Source/SwiftLintFramework/Rules/PrivateUnitTestRule.swift

```diff
@@ -0,0 +1,182 @@
+//
+//  ClassVisibilityRule.swift
+//  SwiftLint
+//
+//  Created by Cristian Filipov on 8/3/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+extension AccessControlLevel {
+    init?(_ dictionary: [String: SourceKitRepresentable]) {
+        guard let
+            accessibility = dictionary["key.accessibility"] as? String,
+            acl = AccessControlLevel(rawValue: accessibility)
+            else { return nil }
+        self = acl
+    }
+}
+
+func superclass(dictionary: [String: SourceKitRepresentable]) -> String? {
+    typealias SKArray = [SourceKitRepresentable]
+    typealias SKDict = [String: SourceKitRepresentable]
+    guard let
+        kindString = dictionary["key.kind"] as? String,
+        kind = SwiftDeclarationKind(rawValue: kindString)
+        where kind == .Class
+        else { return nil }
+    guard let
+        inheritedTypes = dictionary["key.inheritedtypes"] as? SKArray,
+        className = (inheritedTypes[0] as? SKDict)?["key.name"] as? String
+        else { return nil }
+    return className
+}
+
+public class FooTest: NSObject {  }
+
+public struct PrivateUnitTestRule: ASTRule, ConfigurationProviderRule {
+
+    public var configuration: PrivateUnitTestConfiguration = {
+        var configuration = PrivateUnitTestConfiguration(identifier: "private_unit_test")
+        configuration.message = "Unit test marked `private` will be silently skipped."
```

Comment:
What does this mean? `"Unit test marked `private` will be silently skipped."` Skipped from Swift Lint or the actual XCTest will be skipped?